tools/static-assets/scaffolds-js/methods.js

```diff
@@ -0,0 +1,25 @@
+import { Meteor } from 'meteor/meteor'
+import { $$UpperName$$Collection } from './collection';
+
+export const save$$UpperName$$ = async (data) => {
+  return await $$UpperName$$Collection.insertAsync({ ...data });
+}
+
+export const update$$UpperName$$ = async (_id, data) => {
+  return await $$UpperName$$Collection.updateAsync(_id, { ...data });
+}
+
+export const remove$$UpperName$$ = async (_id) => {
+  return await $$UpperName$$Collection.removeAsync(_id);
+}
+
+export const find$$UpperName$$ById = async (_id) => {
+  return await $$UpperName$$Collection.findOneAsync(_id);
+}
```

Comment:
1. Are these "officially recommended" names? _(I personally prefer `create` instead of `save` only because it has the same length as `remove` and `update`.)_
2. Have you thought about using `async function`s instead? _(Again, personal preference, but Meteor codebase is highly scattered in this manner. They also could use `this` for the method context.)_
3. Do you think there's a need to `return` in `update` and `remove`? I know what these functions do and I use their returned values rarely.
4. Have you thought about adding validation here? They're exposed via `Meteor.methods`, so...

Reply:
I agree with points 2 and 4 🙌🏻

Reply:
> I personally prefer create instead of save only because it has the same length as remove and update.)
Now you have triggered me in a good way :), I've never thought about this! 
After some thought, I would agree. In other frameworks, they call it by create as well in reference to the CRUD acronym


what do you mean by:
> Have you thought about adding validation here? They're exposed via Meteor.methods, so...
adding some validation with the check package?

Reply:
Also giving some more tought on to it. Since I'm exporting this using module I could make method.js like this:

```js

import { Meteor } from 'meteor/meteor';
import { CustomerCollection } from './collection';

export function create(data) {
  return CustomerCollection.insertAsync({ ...data });
}

export function update(_id, data) {
  return CustomerCollection.updateAsync(_id, { ...data });
}

export function remove(_id) {
  return CustomerCollection.removeAsync(_id);
}

export function findById(_id) {
  return CustomerCollection.findOneAsync(_id);
}

Meteor.methods({
  'Customer.create': create,
  'Customer.update': update,
  'Customer.remove': remove,
  'Customer.find': findById
});

```

because today the use of those methods along the module import would be:

```js
import { Customer } from './customer';


Customer.findCustomerById('someId')
```
using the code above would look like this:

```js
import { Customer } from './customer';


Customer.findById('someId')
Customer.create({...data})
Customer.someOtherMethod()
```

Reply:
Then if you'd export all of the names "unprefixed" (including the collection object itself), the following code would work:

```ts
import * as Customer from './customer';

Customer.collection;
Customer.create({ ... });
// And so on.
```

But again, it's a matter of preference and what's "officially recommended".

Reply:
Also, I forgot to answer you 🤦 Thank you @faburem for reminding me!

> adding some validation with the check package? or something else?

I'd say `check` is enough.

Reply:
I agree, check would be good enough just to have something in place which is showing the best practice in this regard.